Refactor FAQ Section (#6757)
w3f/polkadot-wiki: docs/general/faq.md

---

!!!info
    This FAQ focuses on technically oriented questions for users interested in certain aspects of
    Polkadot. If you have a more general question related to usage, you may search for the answer on our support
    [Knowledge Base](https://support.polkadot.network/support/home) or the main
    [Polkadot network FAQ](https://polkadot.network/faq).

!!!info "JAM FAQ"
    For questions regarding JAM and its relation to Polkadot, visit the [JAM FAQ](../learn/learn-jam-faq.md)

## Polkadot

### What is the minimum amount of KSM / DOT I can have in my account?

Please see information about [Existential Deposits](../learn/learn-accounts.md#existential-deposit-and-reaping).

### Polkadot Roadmap

For more information on the Polkadot roadmap please visit the
[official Polkadot website](https://www.opengov.watch/polkadot-roadmap).

### Polkadot Launch

> The Genesis block of the Polkadot network was launched on May 26, 2020 at 15:36:21 UTC, as a Proof

Reviewer: ```suggestion
The Genesis block of the Polkadot network was produced on May 26, 2020 at 15:36:21 UTC, as a Proof
```

On August 21, 2020, Redenomination of DOT occurred. From this date, one DOT (old) equals 100 new
DOT.

### Why Polkadot targets 1000 validators while other projects have hundreds of thousands?

> Polkadot's goal to have 1000 validators is set to be something that is practically achievable with high confidence of good performance in a live environment. Furthermore, validators in Polkadot are not the only stakers, and if we consider the number of stakers that can be possible on Polkadot the number can scale up to hundreds of thousands. Since validators are performing critical consensus work to maintain the security of the chain including all of its shards, a more modest number of validators is estimated to start. Upon improvements, such as implementing signature aggregation for finalization messages, the number of validators can reasonably scale up.

Reviewer: ```suggestion
Polkadot's goal to have 1000 validators is set to be something that is practically achievable with high confidence of good performance in a live environment. Furthermore, validators in Polkadot are not the only stakers, and if we consider the number of stakers that can be possible on Polkadot the number can scale up to hundreds of thousands.  Upon improvements, such as implementing signature aggregation for finalization messages, the number of validators can reasonably scale up.
```

> Additionally, other projects sometimes have a different definition of _validator_ that approximates more closely to remote signing keys without the full operation of a validating node. On Polkadot, each validator is running their own validating node and performing full verification of the Relay Chain, voting on finality, producing blocks in their decided slots, and verifying parachain state transitions. Other projects may consider validators and "validating nodes" as separate entities.

Reviewer: ```suggestion
Additionally, other projects sometimes have a different definition of _validator_ that approximates more closely to remote signing keys without the full operation of a validating node. On Polkadot, each validator runs their own validating node and verifies the relay chain, voting on finality, producing blocks in their decided slots, and verifying parachain state transitions. Other projects may consider validators and "validating nodes" as separate entities.
```

Finally, individuals may participate in the block production process indirectly by
[nominating](../learn/learn-nominator.md) validators. In this way, individuals who are not running a
node can still share in staking rewards.

### What is the block time of the relay chain?

Both the Kusama and Polkadot networks are currently operating at a rate of one block every six
seconds.

> ### Does Polkadot have smart contracts?

Reviewer: ```suggestion
### Does Polkadot support smart contracts?
```

> While the Polkadot relay chain does not implement smart contracts directly, [there are numerous

Reviewer: ```suggestion
Polkadot will support Solidity smart contracts by late 2025. Smart contracts deployment is already possible on Kusama and testnets. There are also numerous parachains that support EVM and Wasm smart contracts.
```

Author: I am a bit wary of including this, due to its time dependent nature, which we're looking to avoid in general. However, because the nature of an FAQ usually changes with time, maybe it's an exception? cc @filippoweb3

Reviewer 2: Yeah given the importance of this feature I would add it

Author: In that case, will add it, thank you!

> parachains that do](). It's possible for parachains to enable smart contract functionality and then

Reviewer: ```suggestion
 It's possible for parachains to enable smart contract functionality and then
```
